Gemfile

```diff
@@ -161,4 +164,7 @@ group :test do
 
   # Control time
   gem 'timecop'
+
+  # Modify your ENV
+  gem 'climate_control'
```

Comment:
Don't think we need this. 
Add `state_change_slack_url: replace_me` to `config/settings/production.yml`. Then add the key and value pair to the production secrets

Reply:
Ah, that makes sense. Thanks. Changes pushed.